device_services/meta/diagnostic.md

```diff
@@ -0,0 +1,58 @@
+# Diagnostic Service
+
+This service allows for the control of diagnostic functionalities of associated Things. A Thing can expose a diagnostic API that enables logging of device-specific data (e.g., logging communication between the Thing and the auxiliary devices it uses or controls).
+
+## Service Name
+
+`diagnostic`
+
+## Interfaces
+
+| Type | Interface                  | Value type | Properties        | Description                                                                                                                                                                                  |
+|------|----------------------------|------------|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| out  | cmd.log.start              | null       | `tunnel_protocols` | Commands the Adapter to start logging device related data. If log should contain a data tunneled from auxiliary device protocol must be specified. see [`sup_protocols`](#service-properties) |
```

Comment:
singular form used in definition section,

Reply:
Could you clarify what should be changed?

Reply:
property is "tunnel_protocol" not "tunnel_protocols".

Reply:
fixed